lib/widget/form_helper.dart

```diff
@@ -42,6 +53,19 @@ class FormFieldController extends WidgetController {
       'iconSize': (value) => iconSize = Utils.optionalInt(value),
       'iconColor': (value) => iconColor = Utils.optionalInt(value),
       'fontSize': (value) => fontSize = Utils.optionalInt(value),
+      'boderColor': (color) =>
+      borderColor = Utils.getColor(color) ?? borderColor,
+      'focusedBoderColor': (color) =>
+      focusedColor = Utils.getColor(color) ?? focusedColor,
+      'enabledBoderColor': (color) =>
+      enabledColor = Utils.getColor(color) ?? enabledColor,
+      'boderRadius': (w) => boderRadius = Utils.optionalDouble(w),
+      'focusedBoderRadius': (w) => focusedBoderRadius = Utils.optionalDouble(w),
+      'enabledBoderRadius': (w) => enabledBoderRadius = Utils.optionalDouble(w),
+      'hintStyle': (style) => hintStyle = Utils.getTextStyle(style),
+      'labelStyle': (style) => labelStyle = Utils.getTextStyle(style),
+      'cursorColor': (color) =>
```

Comment:
Don't expose this. If anything the cursor color should be global to all input widgets. Individual widget should not be able to change this.